FUNDING.yml

```diff
@@ -1 +1,2 @@
 custom: https://etherscan.io/address/0x70CCBE10F980d80b7eBaab7D2E3A73e87D67B775
+custom: https://giveth.io/project/vyper
```

Comment:
i think custom should be a list - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository